index.html

```diff
@@ -854,8 +854,9 @@ <h3>Contexts</h3>
 When two software systems need to exchange data, they must use terminology and
 a protocol that both systems understand. As an analogy, consider how two
 people communicate. Both people must use the same language and the words they
-use must mean the same thing to each other. This specification uses the
-<code>@context</code> <a>property</a> to express the context of a conversation.
+use must mean the same thing to each other. The <code>@context</code>
```

Comment:
I think this would be more clearly testable if it said something like this:

"A verifiable credential MUST have a @context property. The @context property is used to express the context of a verifiable credential or a verifiable presentation."

Reply:
Fixed (by you, in your PR).